src/community/ogcapi/dggs/dggs-core/src/main/java/org/geotools/dggs/ZoneWrapper.java

```diff
@@ -73,13 +69,7 @@ public static CoordinateSequence includePole(
             GeometryFactory gf, CoordinateSequence cs, boolean north) {
         List<Coordinate> coordinates = new ArrayList<>(Arrays.asList(cs.toCoordinateArray()));
         Collections.sort(
-                coordinates,
-                new Comparator<Coordinate>() {
-                    @Override
-                    public int compare(Coordinate o1, Coordinate o2) {
-                        return (int) Math.signum(o1.x - o2.x);
-                    }
-                });
+                coordinates, (Coordinate o1, Coordinate o2) -> (int) Math.signum(o1.x - o2.x));
```

Comment:
Good. Hum... just wondering, does it need to be ``(Coordinate o1, Coordinate o2)``, or can it be simply ``(o1, 02)``?

Reply:
It can be `(o1,o2)`, and have updated it accordingly.